src/main/java/nl/workingtalent/wtacademy/user/UpdateUserDto.java

```diff
@@ -0,0 +1,50 @@
+package nl.workingtalent.wtacademy.user;
+
+public class UpdateUserDto {
+
+	private String firstName;
+	private String lastName;
+	private String email;
+	private String password;
+	private Role role;
+
```

Comment:
Deze class is identiek aan CreateUserDto toch?

Reply:
Scherp! Ik weet niet of er nog unieke variables oid worden toegevoegd aan de create DTO. Dus is het goed om de twee dto's te scheiden? Ook in de zin van readability?